.github/workflows/swift-toolchain.yml

```diff
@@ -1134,15 +1144,19 @@ jobs:
           variant: sccache
 
       - uses: nttld/setup-ndk@v1
+        if: ${{ !(inputs.build_os == 'Windows' && inputs.build_arch == 'arm64') }}
```

Comment:
I think this (and elsewhere) could be simplified with `matrix.os == 'Android'`. I am doing something similar for the Mac build to exclude the NDK flag too. @compnerd WDYT?

Reply:
Done.